src/Common/tests/CoreFx.Private.TestUtilities/System/PlatformDetection.Unix.cs

```diff
@@ -46,7 +46,8 @@ public static partial class PlatformDetection
         public static bool IsMacOsHighSierraOrHigher => IsOSX && (m_osxProductVersion.Value.Major > 10 || (m_osxProductVersion.Value.Major == 10 && m_osxProductVersion.Value.Minor >= 13));
         public static bool IsNotMacOsHighSierraOrHigher => !IsMacOsHighSierraOrHigher;
         public static bool IsMacOsMojaveOrHigher => IsOSX && (m_osxProductVersion.Value.Major > 10 || (m_osxProductVersion.Value.Major == 10 && m_osxProductVersion.Value.Minor >= 14));
-        
+        public static bool IsMacOsCatalinaOrHigher => IsOSX && (m_osxProductVersion.Value.Major > 10 || (m_osxProductVersion.Value.Major == 10 && m_osxProductVersion.Value.Minor >= 15));
```

Comment:
we may need to add to refs and Windows => false.

Reply:
interestingly this actually builds correctly on my Windows without that change but will add for consistency with the rest

Reply:
Looking again, seems we don't do that elsewhere, i.e.:

```
F:\src\corefx4\src\Common\tests\CoreFx.Private.TestUtilities>findstr /sinpc:"IsMacOsMojaveOrHigher" *.cs
System\PlatformDetection.Unix.cs:48:        public static bool IsMacOsMojaveOrHigher => IsOSX && (m_osxProductVersion.Value.Major > 10 || (m_osxProductVersion.Value.Major == 10 && m_osxProductVersion.Value.Minor >= 14));
```